lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -396,9 +396,19 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
   template <typename TGroupOp, typename TCalyxLibOp, typename TSrcOp>
   LogicalResult buildLibraryOp(PatternRewriter &rewriter, TSrcOp op,
                                TypeRange srcTypes, TypeRange dstTypes) const {
+    auto castToInteger = [](Type type) -> Type {
```

Comment:
```suggestion
    /// Performs a bit cast from the floating point value to a signless integer type. Calyx treats everything as bit vectors, and leaves their interpretation to the respective operation using it. In CIRCT Calyx, we use `IntegerType` to represent a bit vector.
    auto castToInteger = [](Type type) -> Type {
```

Reply:
I would add this comment to make it clear. I think this function would be useful in the `calyx_utils` file too, since other folks will likely use it.